Formula/w/wcstools.rb

```diff
@@ -0,0 +1,21 @@
+class Wcstools < Formula
+  desc "Tools for using World Coordinate Systems (WCS) in astronomical images"
+  homepage "http://tdc-www.harvard.edu/wcstools/"
+  url "http://tdc-www.harvard.edu/software/wcstools/wcstools-3.9.7.tar.gz"
+  sha256 "525f6970eb818f822db75c1526b3122b1af078affa572dce303de37df5c7b088"
+  license "GPL-2.0-or-later"
+
+  def install
+    system "make", "all"
+    rm_r(Dir.glob("bin/*.dSYM")) if OS.mac?
+
+    prefix.install "bin"
+    bin.install "wcstools"
+    include.install Dir["libwcs/*.h"]
+    man1.install Dir["man/man1/*.1"]
```

Comment:
```suggestion
    (include/"libwcs").install buildpath.glob("libwcs/*.h")
    man1.install buildpath.glob("man/man1/*.1")
```
Do we really need to install these headers? We're not installing the library, so they don't seem very useful.

Reply:
No, There is no need for these headers as it may lead to conflict with `wcslib`. if the version of `wcslib` is difference
wcstools should worked without the headers.